### ISSUE TITLE
ControlStructureSpacingSniff and alternative syntax for control structures

### ISSUE THREAD

**Issue by EvgenyOrekhov** [slevomat/coding-standard#508]
```php
<?php if ($a): ?>
    123
<?php endif; ?>

```

Produces the following warnings:

```text
 1 | ERROR | [x] Expected 1 lines before "if", found -1.
   |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure)
 1 | ERROR | [x] Expected 1 lines after "if", found -1.
   |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
```

1. Is alternative syntax supported?
2. Is it possible to make ControlStructureSpacingSniff ignore alternative syntax?

Slevomat Coding Standard version: 4.8.3
PHP_CodeSniffer version: 3.3.2
PHP version: 7.2.8

**Comment by kukulich** [comment]
1. No
2. If someone sends PR

But it really doesn't make sense to check this type of code.

**Comment by EvgenyOrekhov** [comment]
@kukulich The "views" of the MVC pattern are a big part of our codebases, we check all our views with linters and static analyzers. Since Slevomat Coding Standard does not support views and since it is impractical to manually add every view file to exclusion list, we just have to disable some Slevomat rules completely, which is a shame.

**Comment by ondrejmirtes** [comment]
@EvgenyOrekhov Can you recognize view files based on a different file extension or something like that? BTW you should switch to a template engine like Twig or Latte.

**Comment by EvgenyOrekhov** [comment]
@ondrejmirtes Great idea, thanks! We actually have them under `views` subdirectories, so we should be able to exclude them like that:

```xml
<rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
    <exclude-pattern>*/views/*</exclude-pattern>
</rule>
```

PHP *is* a template engine, it works fine for us =)
